src/routes/controllers/users.controller.js

```diff
@@ -117,3 +121,26 @@ export async function getReadingStats(req, res) {
     return errorResponse(res, 500, error.message);
   }
 }
+
+/**
+ *
+ * @param {Object} req express request
+ * @param {Object} res express response
+ * @returns {Array} user bookmarks
+ */
+export async function getUsersBookmarks(req, res) {
+  const { user } = req;
+
+  try {
+    const currentUser = await getUserbyId(user.id);
```

Comment:
Great work here. I am trying to understand why you are first querying for a user then querying for the user's bookmarks in 2 queries. It would be nice if we can make it one.

Reply:
Bookmark is not a model, hence the include property cannot work.

Reply:
this function will not work too
```Javascript
const usersBookmarks = await Bookmark.findAll({ where: { id: user.id }}); 
```